sdk/core/azure-core/azure/core/pipeline/policies/_distributed_tracing.py

```diff
@@ -75,6 +75,7 @@ class DistributedTracingPolicy(SansIOHTTPPolicy):
 
     def __init__(self, **kwargs):
         self._network_span_namer = kwargs.get('network_span_namer', _default_network_span_namer)
+        self._tracing_attributes = kwargs.get('tracing_attributes', {})
```

Comment:
Please add docstring for "tracing_attributes"

Reply:
fixed